pybaseball/amateur_draft.py

```diff
@@ -1,12 +1,16 @@
 import pandas as pd
 import requests
 
-def get_draft_results(year, round): 
+from .datahelpers import caching
+
+
+def get_draft_results(year, round):
     url = f"https://www.baseball-reference.com/draft/?year_ID={year}&draft_round={round}&draft_type=junreg&query_type=year_round&"
-    res = requests.get(url, timeout=None).content 
+    res = requests.get(url, timeout=None).content
     draft_results = pd.read_html(res)
     return draft_results
 
+@caching.dataframe_cache()
```

Comment:
Can we have a shorter decorator if it's the only one?
Just `@caching.cache()`

Reply:
I think being explicit here is good, especially if we add any other cache types later.

Reply:
If we add later, than rename it I think.

Reply:
even then, just to `df_cache`

Reply:
Why? To save 7 characters?

Reply:
cleaner. I prefer `@caching.cache()`

Reply:
I prefer being explicit over implicit here.